### PR TITLE
Release 1.0.1

### PR DESCRIPTION
**Pull request by aCampello**
Release checklist

- [x] Change `wellcomeml/__version__.py`
- [x] Add changelog
- [x] `make dist`
- [x] Verify new package was generated correctly on the [pip registry](https://pypi.org/project/wellcomeml)
 and GitHub releases
- [x] "Yank" CalVER versions on pyPi registry
- [x] Make sure `pip install wellcomeml` install correct version

Fixes https://github.com/wellcometrust/WellcomeML/issues/160

### CODE REVIEW

File: wellcomeml/__version__.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 __name__ = "wellcomeml"
-__version__ = "2021.2.1"
+__version__ = "1.0.0"
```
Comment (lizgzil) [inline wellcomeml/__version__.py:2]: 🎉

**Review by nsorros — APPROVED** [review]
LGTM, merge when you tick all the boxes

**Comment by aCampello** [comment]
`make dist` works uneventfully

**Comment by aCampello** [comment]
Pypi complained about 1.0.0 

```bash
HTTPError: 400 Bad Request from https://upload.pypi.org/legacy/
This filename has already been used, use a different version. See https://pypi.org/help/#file-name-reuse for more information.
```
Probably something to do with the fact that I used 1.0.0 before for test... investigating

**Comment by aCampello** [comment]
Well this is awkward, seems that PyPI is not enjoying 1.0.0 because there was a test one created in the past. Will figure out if there is anyway to by pass it unless we'll have to be 1.0.1 or something!

**Comment by aCampello** [comment]
Ok, for a number of reasons it seems that we won't be able to release 1.0.0 on pypi, since in the past there was a 1.0.0 deleted.

https://github.com/pypa/packaging-problems/issues/74

**Comment by nsorros** [comment]
> Ok, for a number of reasons it seems that we won't be able to release 1.0.0 on pypi, since in the past there was a 1.0.0 deleted.
> 
> [pypa/packaging-problems#74](https://github.com/pypa/packaging-problems/issues/74)

Yep, releases are not mutable. If they were that would be a problem since one day a library you installed by pinning a version worked and the other it did not out of no fault of your own. 1.0.1 it is.

In the future btw if you want to test something, PyPi recommends doing it in a test account. I also have a test account which I used when we first released wellcomeml

**Comment by aCampello** [comment]
Yes, that was mea culpa. I was too sloppy with the test.

**Comment by aCampello** [comment]
Running make dist again....

**Comment by aCampello** [comment]
Pip install outputs:
```
> pip freeze | grep wellcomeml
wellcomeml==1.0.1
```

**Comment by aCampello** [comment]
Trying to pip install old versions now. Pip installing pinned versions work:

```
pip install wellcomeml==2020.9.0 --no-cache-dir
```

Returns

```
pip freeze | grep wellcomeml
wellcomeml==2020.9.0
```

Note that assignments like `wellcomeml < 2020.9.0` won't work. User will have to pin the exact version.

**Comment by aCampello** [comment]
Incidentally, this PR fixes https://github.com/wellcometrust/WellcomeML/issues/160

**Pull request merged**